### DISCUSSION TITLE
Progress Callback?

### DISCUSSION THREAD

**Discussion by albrrt** [dobrado76/Stable-Diffusion-Unity-Integration#5] [Ideas]
Hi @dobrado76 I am working on a solution to prevent Editor from hanging while waiting for an image to return. 

See https://github.com/albrrt/Stable-Diffusion-Unity-Integration/commit/0187e8ec220ddb69e9116e893fca417412a39949

It would be good to add some form of progress callback to this, so we don't toggle between windows to check progress.
 
Would this involve making a separate call to the `/sdapi/v1/progress` endpoint? 

I'd be interested to hear your thoughts

**Comment by dobrado76** [comment]
Good question. I will look into that. I'm not sure, let give it a try.
